### PR TITLE
Old impl fixes

### PR DESCRIPTION
**Pull request by kikkomep**
This PR aims to fix some issues of the very first API implementation

### CODE REVIEW

File: interaction_experiments/WfHub2LmInteractionExample.ipynb [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# get userinfo from Workflow Hub\n",
+    "user_info = s.get(os.path.join(wfhub_url, 'people/current'))\n",
```
Comment (simleo) [inline interaction_experiments/WfHub2LmInteractionExample.ipynb:70]: Just a note that this and other WorkflowHub API calls, when testing things with self-signed certificates, need the `verify=False` param setting. E.g.:

```python
s.get(os.path.join(wfhub_url, 'people/current'), verify=False)
```

File: lifemonitor/utils.py [hunk 2]
```diff
@@ -8,6 +8,7 @@
 import requests
 
 RO_CRATE_METADATA_FILENAME = "ro-crate-metadata.jsonld"
+RO_CRATE_TEST_DEFINITION_FILENAME = "test-suite-definition.json"
```
Comment (simleo) [inline lifemonitor/utils.py:11]: The metadata file is called `test-metadata.json` in the [current specs](https://github.com/crs4/life_monitor/wiki/Test-Metadata-Draft-Spec).

File: lifemonitor/utils.py [hunk 3]
```diff
@@ -57,7 +58,22 @@ def load_ro_crate_metadata(roc_path):
...
+
+
+def load_test_definition_filename(filename):
+    with open(filename) as f:
+        return json.load(f)
+
+
+def search_for_test_definition(roc_path, ro_crate_metadata: dict):
+    # first search on the root roc_path for a test_definition file
+    filename = os.path.join(roc_path, RO_CRATE_TEST_DEFINITION_FILENAME)
```
Comment (simleo) [inline lifemonitor/utils.py:73]: As per the [current specs](https://github.com/crs4/life_monitor/wiki/Test-Metadata-Draft-Spec), the test metadata file (`test-metadata.json`) is expected under a `test` directory, not at the top level.

File: interaction_experiments/WfHub2LmInteractionExample.ipynb [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# required packages\n",
+    "import os\n",
+    "import requests\n",
+    "from pprint import pp"
```
[the remark this answers is not in the document]
Comment (kikkomep) [reply interaction_experiments/WfHub2LmInteractionExample.ipynb]: Ok! Fixed by https://github.com/crs4/life_monitor/pull/32/commits/3ea89144ff2994d0ca66cc2694ffc755add151f4

File: interaction_experiments/WfHub2LmInteractionExample.ipynb (hunk 1 above)
Comment (kikkomep) [reply interaction_experiments/WfHub2LmInteractionExample.ipynb:76]: Fixed by https://github.com/crs4/life_monitor/pull/32/commits/d74654f9259935b0e4acf4d006feaddfd10fd0d4

File: lifemonitor/utils.py (hunk 2 above)
Comment (kikkomep) [reply lifemonitor/utils.py]: Fixed by https://github.com/crs4/life_monitor/pull/32/commits/1d83f48a4df8fee2d163eff204d3d9f2f58e9e80

File: lifemonitor/utils.py (hunk 3 above)
Comment (kikkomep) [reply lifemonitor/utils.py]: Fixed by https://github.com/crs4/life_monitor/pull/32/commits/351ff8e8cc14778d738a07fe3259770ef53e755c

**Pull request merged**